projects/storefrontlib/cms-components/misc/icon/fontawesome-icon.config.ts

```diff
@@ -45,9 +45,15 @@ export const fontawesomeIconConfig: IconConfig = {
       SORT_DOWN: 'fas fa-sort-down',
       SORT: 'fas fa-sort',
       EXPAND_ARROWS: 'fas fa-expand',
+      CLIPBOARD_LIST: 'fas fa-clipboard-list',
```

Comment:
If adding them is breaking change, maybe augment this config in asm.

Reply:
I don't believe this is a breaking change.  We are only adding new values to the configuration, so no previously used values would be changed.  The configurations are deep merged into one another, so any conflicting values provided by the merchant would be overridden.